sdk/entra/WebJobs.Extensions.AuthenticationEvents/src/WebJobs.Extensions.AuthenticationEvents/ConfigurationManager.cs

```diff
@@ -0,0 +1,66 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.Globalization;
+using System.Linq;
+
+namespace Microsoft.Azure.WebJobs.Extensions.AuthenticationEvents
+{
+    internal class ConfigurationManager
+    {
+        private readonly string[] SERVICE_IDs = new string[] { "99045fe1-7639-4a75-9d4a-577b6ca3810f" };
+        internal const string OPENID_CONNECTHOST = "https://login.microsoftonline.com";
+        private const string BYPASS_VALIDATION = "AuthenticationEvents__BypassTokenValidation";
+        private const string CUSTOM_CALLER_APPID = "AuthenticationEvents__CustomCallerAppId";
+        internal const string TENANT_ID = "AuthenticationEvents__TenantId";
+        internal const string AUDIENCE_APPID = "AuthenticationEvents__AudienceAppId";
+        internal const string TOKEN_V1_ISSUER = "https://sts.windows.net/{0}/";
```

Comment:
Is this valid across all clouds?

Reply:
As above.. I will get back to you on this...

Reply:
I have made a few changes here to support multiple clouds, once I get the service info from Product/Eng in Redmond will update, but for now only public cloud.